.github/workflows/board-automation.yaml

```diff
@@ -0,0 +1,17 @@
+name: Stripey
+
+on:
+  issues:
+    types:
+      - opened
+      - reopened
+jobs:
+  add_to_project:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: skeet70/default-project-board-action@v1
```

Comment:
For actions not from `actions/` (i.e. unofficial), we should pin to a revision so they can't change it and expose secrets.

Reply:
Sweet!
I didn't know about it. Thanks for sharing @mook-as 
I'll fix that.